modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-3-4.adoc
+
+[id="migration-network-traffic-redirection-strategies_{context}"]
+= Network traffic redirection strategies
+
+After a successful migration, you have to redirect network traffic of your stateless applications from the source cluster to the target {product-title} {product-version} cluster.
+
+All strategies for redirecting network traffic described below are based on these assumptions:
+
+* Applications are deployed on the {product-title} {product-version} cluster. The original application pods are still running on the source cluster.
```

Comment:
Change "the" to "a". This is an assumption, not a certainty.

Remove "original". It is unnecessary because you mention the source cluster.

Also, I think this should be joined with "and" because this is one assumption. Suggestion: 

"The application pods are running on the source and target clusters."